Model/Feed.php

```diff
@@ -76,19 +76,16 @@ public function __construct(
 
     /**
      * Process the product feed and update the progress file, in page sizes
-     * of 1 by default, speed gains for higher batches were negligible vs 
+     * of 1 by default, speed gains for higher batches were negligible vs
```

Comment:
Comment is out of date. I think page sizes > 50 didn't show any further speed gains